ui/desktop/app/components/credentials-panel/index.hbs

```diff
@@ -61,7 +62,7 @@
 
       <ul class='credential-secret'>
         {{#if credential.secrets}}
-          {{#each credential.secrets as |secret|}}
+          {{#each (order-by credential.secrets 'username') as |secret|}}
```

Comment:
Shouldn't username and password be next to each other? How do we handle if a user has many different keys and username and password might be far apart from each other?

Should we make this more generic to handle any arbitrary sorting order?

Reply:
that's a valid point, looking into this

Reply:
changed the function logic to follow the custom order array we pass. So now, the objects will be ordered based on whatever custom order is passed to the function. Let me know what you think